radixdlt/babylon-wallet-android
# Fixes regarding showing the claim screen for existing users

## Description
The problem was that existing users didn't have the newly introduced device info saved in preferences. So when a backup were first created, it used the `profile.header.lastUsedOnDevice.id` which is an id not stored in device, thus considered different in the new cloud backup system. The solution is to
1. Make sure existing users always update the `lastUsedOnDeviceInfo` on each save profile. Since the device info is consistent and stored in preferences, it will not change upon each save.
2. When the user sees the migration to new backup system screen, and sing in to the new backup system, force-resave the profile (by resetting the `isCloudProfileSyncEnabled = true`) so as the profile is updated with the new device info and the sync process is started.


## How to test
1. Install the latest live version from tag `1.5.4`
2. Create a wallet and backup with system backups
3. Then install this branch. 
4. Whatever you do this should make sure that you will not see the claimed by another device screen

## PR submission checklist
